yarn.lock

```diff
@@ -2207,30 +2207,64 @@
     "@dhis2/app-service-data" "3.2.4"
     "@dhis2/app-service-offline" "3.2.4"
 
+"@dhis2/app-runtime@^3.2.5":
```

Comment:
There's two versions of app-runtime in this file so it needs to be deduplicated with `npx yarn-deduplicate yarn.lock`

Reply:
This would be a good opportunity to bump `cli-app-scripts`, whose latest version checks for this automatically

Reply:
You're right, good catch. Should be good now.

Reply:
> This would be a good opportunity to bump cli-app-scripts, whose latest version checks for this automatically

Ah yeah I can do that. Will do that now

Reply:
Updated cli-app-scripts in 998e530